modules/phase_field/tests/DeformedGrain/DeformedGrain.i

```diff
@@ -0,0 +1,122 @@
+#This example shows the growth of a recrystallized grain (gr1) at the expense of a deformed grain (gr0)
+#Note that without deformation No growth would take place since a flat interface does not move.
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 32
+  ny = 0
+  nz = 0
+  xmin = 0
+  xmax = 64
+[]
+
+[GlobalParams]
+  block = 0
+  op_num = 2
+  ndef = 1 # Number of deformed grains.
```

Comment:
Comment is better than no comment, but I strongly suggest you rename the parameter. Unnecessarily brief names introduce the chance of clashes in the future. I would suggest `deformed_grain_num`